packages/standalone/index.ts

```diff
@@ -0,0 +1 @@
+export { Web3Modal } from './src/client'
```

Comment:
Since this is the index for the package, maybe it'd be nice to also add

```suggestion
export { Web3Modal } from './src/client'
export { Web3Modal as default } from './src/client'
```

Reply:
Keeping it consistent / one way to import for now.